src/Core/src/Platform/Android/MauiScrollView.cs

```diff
@@ -122,6 +123,24 @@ public void SetOrientation(ScrollOrientation orientation)
 			}
 		}
 
+		public override void Draw(Canvas? canvas)
```

Comment:
@jsuarezruiz this method is taking up 21.07ms on startup now:

![image](https://user-images.githubusercontent.com/840039/159529589-58299857-cf37-4a4c-9bc0-dbc3f6709122.png)

I'm having trouble following what this fixes, is there a link to an issue somewhere? I want to see if there is an alternative to overriding `Draw()` to solve this problem. Or maybe we could do it all on the Java side?

Here is the `dotnet trace` output: [maui-scrollview.zip](https://github.com/dotnet/maui/files/8326246/maui-scrollview.zip)